### PR TITLE
No random Highlander

### PR DESCRIPTION
**Pull request by Bod9001**
scrubbers can now be adjusted from prefab
### Changelog:

CL: [Fix] Stopped Highlander event from being randomly triggered

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs [hunk 1]
```diff
@@ -35,11 +35,16 @@ public override void OnEventStart()
...
 		{
+			int Round = GameManager.RoundID;
 			remainingTime = eventTime;
 			while (remainingTime > 0f)
 			{
 				remainingTime -= 1f;
 				yield return WaitFor.Seconds(1f);
+				if (Round != GameManager.RoundID)
+				{
+					yield break;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs:46]: wouldn't this escape the while loop but still run the next line, which is ending the round?

**Comment by corp-0** [comment]
can you register the other changes in this PR? What are they about?

**Comment by MaxIsJoe** [comment]
But I like how nobody expected the random Scottish inquisition :(

File: UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs:46]: yield break 
I think breaks out of the entire thing

Comment (corp-0) [reply UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventHighlander.cs:46]: okay

**Pull request merged**